tests/packagedcode/test_package_instance.py

```diff
@@ -37,6 +37,13 @@ def test_package_instance_scan_python_with_uuid(self):
         run_scan_click(['--package', '--strip-root', '--processes', '-1', test_dir, '--json', result_file])
         check_json_scan(expected_file, result_file, regen=False, ignore_instance_uuid=True)
 
+    def test_package_instance_scan_python_with_test_manifests(self):
+        test_dir = self.get_test_loc('instance/pypi-with-test-manifests')
+        result_file = self.get_temp_file('json')
+        expected_file = self.get_test_loc('instance/python-package-instance-expected-with-test-manifests.json')
+        run_scan_click(['--package', '--strip-root', '--processes', '-1', test_dir, '--json', result_file])
+        check_json_scan(expected_file, result_file, regen=True, remove_instance_uuid=True)
```

Comment:
Hey @pombredanne I left `regen=True` by mistake could you add a small commit in your release PR (I could do this too, but not sure whether you have unpushed commits/work there)

Reply:
@AyanSinhaMahapatra do not wait for me... please go ahead and merge alright as soon as things are working OK :)